### PR TITLE
Updates the README and Bug/Documentation reporting templates

### PR DESCRIPTION
**Pull request by mineminemine**
Fixes some typos, grammar errors and whatnot

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -4,43 +4,43 @@ labels: ["bug"]
...
+      description: "A concise description of what you're experiencing. Please include a screenshot or screen recording if possible."
       placeholder: "Describe the current behavior..."
     validations:
       required: true
   - type: dropdown
     id: expected_behavior
     attributes:
       label: Expected Behavior
-      description: "What behavior would you expect to see if the bug was fixed?"
+      description: "What behaviour would you have expected to see?"
```
Comment (Cupcak3) [inline .github/ISSUE_TEMPLATE/bug-report.yml:26]: I would ask that American English preferred spelling be used for words.

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -70,43 +70,43 @@ body:
     id: genshin_version
     attributes:
       label: Genshin Impact Version
-      description: Which Genshin Impact version did this issue happen?
-      placeholder: Please enter the Genshin Impact version (i.e. 2.8)..."
+      description: In which Genshin Impact version did this issue happen?
```
Comment (Cupcak3) [inline .github/ISSUE_TEMPLATE/bug-report.yml:73]: Description should be something like "Which Genshin Impact version does this issue affect?"

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 3]
```diff
@@ -70,43 +70,43 @@ body:
...
+      placeholder: Please enter the Genshin Impact version (i.e., 2.8)..."
     validations:
       required: true
   - type: input
     id: scanner_version
     attributes:
       label: Inventory Kamera Version
-      description: Which Inventory Kamera are you using?
-      placeholder: Please enter the Inventory Kamera version (i.e. 1.2.7)..."
+      description: Which Inventory Kamera version did you use?
```
Comment (Cupcak3) [inline .github/ISSUE_TEMPLATE/bug-report.yml:81]: The original description is fine or something like "Which version of Inventory Kamera is affected?"

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 4]
```diff
@@ -70,43 +70,43 @@ body:
...
+      placeholder: Please enter the Inventory Kamera version (i.e., 1.2.7)..."
     validations:
       required: true
   - type: input
     id: screen_resolution
     attributes:
       label: Screen resolution, screen mode, and UI scale
-      description: What screen resolution(s) are you using?
+      description: What screen resolution(s), screen mode, and UI scale are you using?
```
Comment (Cupcak3) [inline .github/ISSUE_TEMPLATE/bug-report.yml:89]: Original description is fine

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 5]
```diff
@@ -115,7 +115,7 @@ body:
       label: Preflight Checklist
       description: Checklist before sending the bug report.
       options:
-        - label: "I have checked the issue page, and there were none that similar to the one I'm reporting."
+        - label: "I have checked the issue page, and there are none that similar to the one I'm reporting."
```
Comment (Cupcak3) [inline .github/ISSUE_TEMPLATE/bug-report.yml:118]: Better description "I have checked for similar issues and found nothing similar"

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 6]
```diff
@@ -125,9 +125,9 @@ body:
           required: false
         - label: "I have attached the error log to the bug report."
           required: false
-        - label: "I am available to later explains the issue."
+        - label: "I will be available to explain the issue."
```
Comment (Cupcak3) [inline .github/ISSUE_TEMPLATE/bug-report.yml:128]: This field doesn't even need to exist

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 7]
```diff
@@ -125,9 +125,9 @@ body:
...
           required: false
-        - label: "I did not enable any image filtering software or plugins (ie. HDR, Image Sharpening, etc.) while scanning."
+        - label: "I did not enable any image filtering software or plugins (i.e. HDR, Image Sharpening, etc.) while scanning."
```
Comment (Cupcak3) [inline .github/ISSUE_TEMPLATE/bug-report.yml:130]: Better description "I have disabled any and all screen filtering software or plugins (HDR, Image Sharpening, etc.)"

File: README.md [hunk 8]
```diff
@@ -1,7 +1,7 @@
 # Inventory Kamera - A Genshin Data Scanner
-Fan-made Genshin Impact tool that scans your Characters, Weapons, Artifacts, Materials, and Character Development items in your Inventory using the OCR technique.
+A fan-made Genshin Impact tool that scans your Characters, Weapons, Artifacts, Materials, and Character Development items in your Inventory using the OCR technique.
```
Comment (Cupcak3) [inline README.md:2]: Unnecessary

File: README.md [hunk 9]
```diff
@@ -1,7 +1,7 @@
...
 
-This scanner exports in `.GOOD`, a JSON-based exporting format, which allows you to use it with compatible online Genshin Impact tools. These tools include artifact optimizing tools including [Genshin Optimizer](https://frzyc.github.io/genshin-optimizer/#/), [SEELIE.me](https://seelie.me/), and [Aspirine's Genshin Impact Calculator](https://genshin.aspirine.su/).
+This scanner exports in `.GOOD`, a JSON-based  export format, which allows you to use it with compatible online Genshin Impact tools. These tools include artifact optimizing tools including [Genshin Optimizer](https://frzyc.github.io/genshin-optimizer/#/), [SEELIE.me](https://seelie.me/), and [Aspirine's Genshin Impact Calculator](https://genshin.aspirine.su/).
```
Comment (Cupcak3) [inline README.md:4]: Unnecessary

File: README.md [hunk 10]
```diff
@@ -32,69 +32,72 @@ To install, unzip the downloaded release to a folder of your choosing and open t
...
+> Due to the way Inventory Kamera is compiled, application settings that you change from default (most importantly, the export folder!) will be reset to their default values. It is recommended to continue to use one folder location when unzipping future releases. More information can be [found here](https://github.com/Andrewthe13th/Inventory_Kamera/issues/301#issuecomment-1179611917).
 
 
 ## Set up with your Genshin Impact
 1. Log in to Genshin Impact and click start.
 2. Open [Paimon menu](https://genshin-impact.fandom.com/wiki/Paimon_Menu). (Default shortcut: `ESC`)
 3. Go to `Settings` (Cog Icon ⚙) and set it to these settings:
-   - Under `Languages`, set *Game Language* to English.
-   - Under `Graphics`, set *Display Mode* to *any windowed resolution* that is *16:9 or 16:10 ratio*. Make sure that the entire game window is visible on the screen.
+   - Under `Languages`, set the *Game Language* to English.
```
Comment (Cupcak3) [inline README.md:42]: Unnecessary

File: README.md [hunk 11]
```diff
@@ -32,69 +32,72 @@ To install, unzip the downloaded release to a folder of your choosing and open t
...
+   - Under `Graphics`, set the *Display Mode* to *any windowed resolution* that is *16:9 or 16:10 ratio*. Make sure that the entire game window is visible on the screen.
```
Comment (Cupcak3) [inline README.md:43]: Unnecessary

File: README.md [hunk 12]
```diff
@@ -32,69 +32,72 @@ To install, unzip the downloaded release to a folder of your choosing and open t
...
      - Example of 16:9 includes 1920x1080 (Full HD), 3840x2160 (4K), 2560x1440 (2K).
      - Example of 16:10 includes 1920x1200.
      - Don't know if a resolution is 16:9 or 16:10? [Find out here](https://andrew.hedges.name/experiments/aspect_ratio/).
      - > **Warning**<br>
-        If you have an ultrawide monitor please see [this thread](https://github.com/Andrewthe13th/Inventory_Kamera/issues/40)
-   - Under `Controls`, set *Control Type* to Keyboard.
-      - If you have rebound the inventory key (default: B) or character screen key (default: C), either revert your binding to default or set up new key binding in Inventory Kamera.
+        If you have an ultrawide monitor, please see [this thread](https://github.com/Andrewthe13th/Inventory_Kamera/issues/40)
+   - Under `Controls`, set the *Control Type* to Keyboard.
+      - If you have rebounded the inventory key (default: B) or character screen key (default: C), either revert your binding to default or set the key binding in Inventory Kamera.
```
Comment (Cupcak3) [inline README.md:50]: Rebound is appropriate. Sentence should probably read something like "If you have rebound a key, make sure to bind the keys appropriately in Inventory Kamera"

File: README.md [hunk 13]
```diff
@@ -32,69 +32,72 @@ To install, unzip the downloaded release to a folder of your choosing and open t
...
 ## How to Configure Inventory Kamera
-Before starting the scanner, you can (optionally) choose to edit these following configurations:
+Before starting the scanner, you can (optionally) choose to edit the following configurations:
 
-- Configure which category (Weapons, Artifacts, Characters, Items) wanted to be scanned.
-- Configure minimum weapon and artifact rarity to be scanned.
-- Set the scanner delays to slow Inventory Kamera's scanning speed if you might not have a strong PC
-- Set the file export destination in the File Directory
+- Configure which category (Weapons, Artifacts, Characters, Items) you want to be scanned.
+- Configure the minimum weapon and artifact rarity to be scanned.
```
Comment (Cupcak3) [inline README.md:57]: Unnecessary

File: README.md [hunk 14]
```diff
@@ -32,69 +32,72 @@ To install, unzip the downloaded release to a folder of your choosing and open t
...
 ## How to Run Inventory Kamera
-Starts the Inventory Kamera scans by **leaving the game screen with the Paimon Menu stays open** and click 'Scan' to start scanning.
+1. Open [Paimon menu](https://genshin-impact.fandom.com/wiki/Paimon_Menu) (Default shortcut: `ESC`) and leave it open.
+2. Open Inventory Kamera and configure the settings as you wish, or else leave them as default.
+3. Start the scan by clicking on the 'Scan Genshin' button. Leave it to scan and do not use your mouse or keyboard in the meantime, as it will interrupt the scanning process.
+4. Once the scan is finished, you can find the exported scan file in the GenshinData folder within the same folder as the application or in the folder that you have set the exports to.
 
 > **Warning**<br>
-> While scanning, **do not use your mouse or keyboard**. We use your keyboard and mouse to automate artifact scanning.
+> While scanning, **do not use your mouse or keyboard**. The scanner uses mouse and keyboard inputs to automate artifact scanning.
```
Comment (Cupcak3) [inline README.md:69]: Remove 'artifact'

**Review by Cupcak3 — CHANGES_REQUESTED** [review]
Some comments on specific line changes. If a change doesn't have a comment it's ok.

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 15]
```diff
@@ -4,43 +4,43 @@ labels: ["bug"]
 body:
   - type: markdown
     attributes:
-      value: "# Bug Report template"
+      value: "# Bug Report"
   - type: markdown
     attributes:
-      value: "Please describe each question as much as you can. This will speed up our bug fixing process."
+      value: "Please describe the bug as best as you can. This will speed up our bug fixing process."
```
Comment (sagelga) [inline .github/ISSUE_TEMPLATE/bug-report.yml:10]: Most of the users don't even understand what the bug is. 

All they want is to enjoy Genshin and use the scanner.

I suggests we choose wording for normal person.

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 16]
```diff
@@ -51,7 +51,7 @@ body:
     id: deviceOS
     attributes:
       label: "Device OS"
-      description: "What operating system are you using? (e.g. Windows, Mac, Linux)"
+      description: "What operating system are you using? (e.g., Windows, Mac, Linux)"
```
Comment (sagelga) [inline .github/ISSUE_TEMPLATE/bug-report.yml:54]: Confused on why the , after e.g.

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 6 above)
Comment (sagelga) [reply .github/ISSUE_TEMPLATE/bug-report.yml:128]: Pretty much to let the user understands that they might have to provide us more information for fixing the issue.

File: .github/ISSUE_TEMPLATE/fallback_template.yml [hunk 17]
```diff
@@ -13,47 +13,47 @@ body:
...
-      placeholder: "Please give us some additional information that you seem fit and could speed up our process. "
+      placeholder: "Please give us any other additional information that could speed up the process. "
   - type: checkboxes
     attributes:
       label: "Preflight Checklist"
-      description: "Select one or more solutions"
+      description: "Select one or more solution(s)"
       options:
-        - label: "I understands that this is not a feature request or enhancement request. That will be done in Discussions tab."
+        - label: "I understand that this is not a feature request or enhancement request. Requests can be made in the Discussions tab."
```
Comment (sagelga) [inline .github/ISSUE_TEMPLATE/fallback_template.yml:48]: Not can in "done in Discussions tab" but a must.

File: .github/ISSUE_TEMPLATE/fallback_template.yml [hunk 18]
```diff
@@ -13,47 +13,47 @@ body:
...
           required: true
-        - label: "I understands that this is not a question, crash report, error report, or a bug report. That will be done in Issues > Bug Report."
+        - label: "I understand that this is not a question, a crash report, an error report, or a bug report. Bug reports can be reported in Issues > Bug Report."
```
Comment (sagelga) [inline .github/ISSUE_TEMPLATE/fallback_template.yml:50]: Same here as prev. line

**Review by sagelga — CHANGES_REQUESTED** [review]

**Pull request closed without merging**